Application/EdFi.Ods.AdminApi/Features/Applications/ApplicationModel.cs

```diff
@@ -25,3 +27,9 @@ public class ApplicationResult
     public string? Key { get; set; }
     public string? Secret { get; set; }
 }
+
+[SwaggerSchema(Title = "Profile")]
+public class Profile
+{
+    public int? ProfileId { get; set; }
```

Comment:
```suggestion
    public int? Id { get; set; }
```

Let's change this to simply `Id`. I thought about making it not nullable - because a Profile always has an Id - but then realized that we'll need it to be nullable in the future when we have the ability POST a new Profile.